packages/zql/src/ivm/view-apply-change.ts

```diff
@@ -12,6 +12,7 @@ import type {SourceSchema} from './schema.ts';
 import type {Entry, Format} from './view.ts';
 
 export const refCountSymbol = Symbol('rc');
+export const idSymbol = Symbol('id');
```

Comment:
Or should this be 
`export const keySymbol = Symbol('key');`

Reply:
I don't really care. This is fine